### ISSUE TITLE
Failed import because of SyntaxError from using await in an f-string

### ISSUE THREAD

**Issue by patjdor** [spaceone-dev/grpc_requests#19]
**Describe the bug**
Importing `grpc_requests` v0.0.9 fails because of invalid syntax in an f-string in the `aio` module when using python 3.6. 

The error when using python 3.6.9
```
$ python3 -c "import grpc_requests"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/rapid/.local/lib/python3.6/site-packages/grpc_requests/__init__.py", line 1, in <module>
    from .aio import AsyncClient, ReflectionAsyncClient, StubAsyncClient, get_by_endpoint as async_get_by_endpoint
  File "<fstring>", line 1
    (await self.service_names())
              ^
SyntaxError: invalid syntax
```

It appears that using `await` in an f-string in python 3.6 is invalid syntax, but not in later versions of python as there is no error when using python 3.8.5 and grpc_requests 0.0.9.

**To Reproduce**
1. Create an environment (e.g. docker container, venv) with python 3.6
2. `pip install grpc_requests==0.0.9`
3. `python3.6 -c "import grpc_requests`

**Expected behavior**
There should not be a syntax error when importing grpc_requests.

**Desktop (please complete the following information):**
 - Python: python 3.6.9
 - OS: docker container running a custom version of ubuntu
 - grpc-requests 0.0.9

**Additional context**
In the context in which I want to use grpc_requests, I cannot update the python version past 3.6. Thus, I cannot use the latest version because of the use of `await` inside an f-string However, there is no issue importing and using v0.0.8.

Relevant python issue: [await expressions in f-strings][1].

[1]: https://bugs.python.org/issue28942

**Comment by wesky93** [comment]
thank you! I will check it!

**Comment by wesky93** [comment]
I fix this issue in #21 
now you can download v0.0.10 in pip on python 3.6!
